src/auto/functions.rs

```diff
@@ -26,6 +25,10 @@ use InputStream;
 use Resource;
 use ResourceLookupFlags;
 use SettingsBackend;
+#[cfg(feature = "unix")]
+use UnixMountEntry;
+#[cfg(feature = "unix")]
+use UnixMountPoint;
```

Comment:
Seems it need `feature = "dox"` too as docs fails on https://travis-ci.org/gtk-rs/gio/jobs/570186591#L631
also usual we don't write `feature="unix"`
```
#[cfg(any(unix, feature = "dox"))]
pub fn unix_mount_get_device_path(mount_entry: &mut UnixMountEntry) -> Option<std::path::PathBuf> {
```

Reply:
An idea why the dox feature condition isn't generated?

Reply:
Because it we missed that when futures was added:
```rust
    #[cfg(feature = "futures")]
    #[cfg(any(feature = "v2_60", feature = "dox"))]
    fn launch_uris_async_future<P: IsA<AppLaunchContext> + Clone + 'static>(
```